### ISSUE TITLE
Nix shell simplification broke yarn flow

### ISSUE THREAD

**Issue by tsusanka** [trezor/connect#732]
The commit 0025fff2 introduced some changes and it fails for `yarn flow`. I have reverted this commit in develop 3f1336bf0b08fd827d0dc039c712f777589a62c3 because it was blocking @szymonlesisz.

See this [job](https://gitlab.com/satoshilabs/trezor/connect/-/jobs/916558526). Reproduced on my NixOS.

**How to reproduce**

1. `nix-shell`
2. `yarn`
3. `yarn flow`

**Notes**
If we now install `flow` natively I guess we need to link so it is taken from nix store and not bin modules? I remembered we did smth like that in Suite but there are no symlinks [there](https://github.com/trezor/trezor-suite/blob/develop/shell.nix) now.

**Comment by prusnak** [comment]
Yeah. Symlinks would do the trick. I thought it's not required to do the symlink in this case because `yarn flow` just runs `flow ...`. And in nix-shell the system-wide installed flow is run instead of the one in the node_modules (because the node_modules/bin path is at the end of PATH env variable). Not sure why `yarn flow` runs the binary from node_modules ...
